src/components/Common/video-card.component.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import formatter from '../../utils/formatter';
+
+const VideoCard = (props) => {
+  const { snippet } = props;
```

Comment:
Same as the other component I would detach the props from the response structure.